docs/specification.yml

```diff
@@ -46,7 +46,7 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/403Forbidden"
-  /v1/submit:
+  /v1/submit/{workflow}:
```

Comment:
Nice find

Reply:
All credit to @blankdots  :D

Reply:
You'd still need to add the specification for that field. You could check the `/v1/workflows/{workflow}` endpoint, there's a `parameters` field.